i18n/locales/zh-Hant.json

```
@@ -24,7 +24,10 @@
   "account_page_title": "帳戶",
   "account_page_update_settings_error": "未能更新帳戶設定。",
   "account_page_upgrade_to_plus": "升級至 Plus",
+  "account_register_error_contact_support": "開始遷移",
```

```suggestion
  "account_register_error_contact_support": "開始轉移",
```
`轉移` is used in migration web ui